public/content/history/index.md

```diff
@@ -3,7 +3,7 @@ title: History and Forks of Ethereum
 description: A history of the Ethereum blockchain including major milestones, releases, and forks.
 lang: en
 sidebarDepth: 1
----
+--- 
```

Comment:
revert please =)

Reply:
Hi @wackerow,

I apologize for the delay in responding. I'll review the necessary changes and get back to you with the updates as soon as possible.